### DISCUSSION TITLE
Regarding the Collection of Documents

### DISCUSSION THREAD

**Discussion by wolono** [fuma-nama/fumadocs#885] [Ideas]
Hello everyone, I had an idea while using Fumadocs.
How to manage and present multiple documents?
For example:
Multiple documents in the docs, below is the demonstration table of contents
```
docs/
├── fumadocs/
│   ├── headless/
│       ├── index.mdx
│       └── meta.json
│   ├── ui/
│       ├── index.mdx
│       └── meta.json
│   └── mdx/
│       ├── index.mdx
│       └── meta.json
├── nextjs/
│   ├── App Router/
│       ├── index.mdx
│       └── meta.json
│   ├── Page Router/
│       ├── index.mdx
│       └── meta.json
├── Turbo/
│   ├── Turborepo/
│       ├── index.mdx
│       └── meta.json
│   ├── Turbopack/
│       ├── index.mdx
│       └── meta.json
└── README.md
```
If it is this mode, a single Root Toggle function becomes difficult to manage.
Can we add a list page like feature to display first level document folders in a book like manner
After accessing a first level folder, display it as a root toggle

## Display method when accessing/docs
![image](https://github.com/user-attachments/assets/2992606c-888a-4d82-a272-c541bc1d4ee0)

## How to display when accessing/docs/fumadocs/ui
![image](https://github.com/user-attachments/assets/840b341a-7b2a-4fc6-b80e-82f740ef6e2d)

**Comment by Skreeauk** [comment]
Can't you just create 2+ docs? You can check the website's code structure. The blog is also using MDX.

For the first screenshot, you can also just see here https://fumadocs.vercel.app/docs. Key point is to not make it an optional catch all. instead of [[...slug]], its [...slug]. Read nextjs docs for more info on the difference.

Fuma's blog doesn't use the DocsLayout, so u just make new DocsLayout for the 2nd doc and pass in a new root toggle specifically for the new docs

**Reply by vikramrojo** [reply]
What would the source.config file look like? I can only add one defineDocs export.

**Reply by fuma-nama** [reply]
You can add more than one, for more details see https://fumadocs.vercel.app/docs/mdx

If you have used anything similar to Contentlayer, it should be pretty straightforward.

**Comment by wolono** [comment]
![image](https://github.com/user-attachments/assets/74ee04ab-86e1-4c17-b110-fa422e36a529)
![image](https://github.com/user-attachments/assets/46761435-6ead-48c6-bfea-e1e6669fdea2)
Although it is simple, it works
Since I cannot write code, I can only implement it according to the documentation guidance

**Comment by Gitstar-OC** [comment]
I guess already so much effort has been done in this docs, and since changing layout of the folder may have a problem, you can just write if you want but for multiple docs you might need to configure some files, you can just ask for help!

**Comment by wolono** [comment]
I don't know how to write code, and if I do it myself, seeking help during the process may trouble many people, although I really want to achieve it through my own efforts.

**Comment by fuma-nama** [comment]
I think your implementation above is good enough, what is your question?

**Reply by wolono** [reply]
https://fumadocs.vercel.app/docs
You have achieved what I wanted, great! How can I have it?

**Reply by fuma-nama** [reply]
You mean the docs index page? Build it, you can use the Cards component from Fumadocs UI.
